mne/tests/test_source_estimate.py

```diff
@@ -261,71 +278,95 @@ def test_stc_arithmetic():
         out.append(a)
 
     assert_array_equal(out[0], out[1].data)
+    assert_array_equal(out[2], out[3].data)
     assert_array_equal(stc.sqrt().data, np.sqrt(stc.data))
+    assert_array_equal(vec_stc.sqrt().data, np.sqrt(vec_stc.data))
 
     stc_mean = stc.mean()
     assert_array_equal(stc_mean.data, np.mean(stc.data, 1)[:, None])
+    vec_stc_mean = vec_stc.mean()
+    assert_array_equal(vec_stc_mean.data,
+                       np.mean(vec_stc.data, 2)[:, :, None])
 
 
 @slow_test
 @testing.requires_testing_data
 def test_stc_methods():
-    """Test stc methods lh_data, rh_data, bin, center_of_mass, resample"""
-    stc = read_source_estimate(fname_stc)
-
-    # lh_data / rh_data
-    assert_array_equal(stc.lh_data, stc.data[:len(stc.lh_vertno)])
-    assert_array_equal(stc.rh_data, stc.data[len(stc.lh_vertno):])
+    """Test stc methods lh_data, rh_data, bin(), resample()
+    """
+    stc_ = read_source_estimate(fname_stc)
+    vec_stc_ = VectorSourceEstimate(np.concatenate((
+                                        stc_.data[:, np.newaxis, :],
+                                        np.zeros((
+                                            stc_.data.shape[0],
+                                            1,
+                                            stc_.data.shape[1]
+                                        )),
+                                    ), 1),
```

Comment:
This would be more readable with one or more of 1) using temporary vars and 2) using the newline trick I mentioned before